pennylane/vqe/vqe.py

```diff
@@ -113,6 +119,44 @@ def wires(self):
         """
         return qml.wires.Wires.all_wires([op.wires for op in self.ops], sort=True)
 
+    def simplify(self):
+        r"""Simplifies the Hamiltonian by combining like-terms.
+
+        Returns:
+            .Hamiltonian
```

Comment:
```suggestion
        r"""Simplifies the Hamiltonian by combining like-terms.

```
Doesn't look like anything is returned here :)